app/javascript/packs/base.jsx

```diff
@@ -63,6 +63,7 @@ window.Forem = {
     contentSelector,
     overlay = false,
     size = 's',
+    onOpen,
```

Comment:
I worked with @aitchiss to add a callback to the modal so that we can attach eventListeners to the buttons within the modal. Without this, the modal was being copied into the `#window-modal` div without the event handlers.